src/libraries/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -395,28 +396,28 @@ private bool AgeCookies(string domain)
             }
             lock (m_domainTable.SyncRoot)
             {
-                foreach (DictionaryEntry entry in m_domainTable)
+                foreach (DictionaryEntry? entry in m_domainTable)
                 {
                     if (domain == null)
                     {
-                        tempDomain = (string)entry.Key;
-                        pathList = (PathList)entry.Value; // Aliasing to trick foreach
+                        tempDomain = (string?)entry?.Key;
+                        pathList = (PathList)entry?.Value!; // Aliasing to trick foreach
```

Comment:
All of these changes related to DictionaryEntry should be reverted.  DictionaryEntry is a struct... making it DictionaryEntry? actually makes it a Nullable<DictionaryEntry>.  Similarly, adding these ? here makes this into null checks at run time.

Reply:
How to fix " error CS8605: Unboxing a possibly null value." in line 399 `foreach (DictionaryEntry? entry in m_domainTable)` ?

Reply:
I found only such sample https://github.com/dotnet/runtime/pull/528/files#diff-e6c8afbe4103a5527844f56a910514ccR82-R88

Reply:
Hmm, and I think that change was a mistake as well. I would write it as follows:
```csharp
                foreach (object? item in m_domainTable)
                {
                    DictionaryEntry entry = (DictionaryEntry)item!;
                    ...
```

Reply:
Fixed.

(It seems the method/file) should be refactored later)